components/Newsfeed/Newsfeed.tsx

```diff
@@ -29,12 +29,18 @@ export default function Newsfeed() {
 
   // Update the filter function
   useEffect(() => {
+    const seenTimestamps = new Set()
     const results = allResults.filter(result => {
-      if (isShowingOrgs && result.type === "org") return true
-      if (isShowingBills && result.type === "bill") return true
-      return false
+      const timestampIdentifier = `${result.timestamp.seconds}-${result.timestamp.nanoseconds}`
```

Comment:
Hopefully a dumb question, but just want to double check - if a user is subscribed to both Bill 123 and Org 456 and Org 456 submits testimony on Bill 123, do the timestamps on both the Bill Notification and the Org Notification originate from the same point - or is the timestamp generated at the time the notification is created (where there could be jitter that misaligns these timestamps)?

Reply:
The Bill Notification is basically a duplicate of the Org Notification if a user is subscribed to both the bill and the organization. So the timestamp would always be the same.